rifServices/src/main/java/org/sahsu/rif/services/rest/WebService.java

```diff
@@ -1388,10 +1388,68 @@ private RIFStudySubmission getRIFSubmissionFromJSONSource(
 			JSONObject rifJobSubmission = jsonObject.optJSONObject("rif_job_submission");
 			if (rifJobSubmission != null) {
 				JSONObject studySelection = rifJobSubmission.optJSONObject("study_selection");
+				JSONObject disease_mapping_study = rifJobSubmission.optJSONObject("disease_mapping_study");
+				JSONObject risk_analysis_study = rifJobSubmission.optJSONObject("risk_analysis_study");
+				String name = "<no name>";
+				String description = "<no description>";
+				
+				if (disease_mapping_study != null) {
+					name = disease_mapping_study.optString("name");
+					description = disease_mapping_study.optString("description");	
+				}
+				if (risk_analysis_study != null) {
+					name = risk_analysis_study.optString("name");
+					description = risk_analysis_study.optString("description");
+				}
+				
+				String riskAnalysisDescription=null;
+				String studyType=null;
```

Comment:
Don't need to set it to null, as that's the default.

Reply:
C habits